dashboard/src/index.html

```diff
@@ -8,12 +8,14 @@
 
   <link href="img/favicon.ico" rel="shortcut icon" type="image/vnd.microsoft.icon" />
   <!-- STYLES -->
-  <link rel="stylesheet" type="text/css" href="css/index.css">
   <!-- build:css lib/css/main.min.css -->
   <link rel="stylesheet" type="text/css" href="components/bootstrap/dist/css/bootstrap.min.css">
   <link rel="stylesheet" type="text/css" href="components/font-awesome/css/font-awesome.min.css">
   <link rel="stylesheet" type="text/css" href="components/rdash-ui/dist/css/rdash.min.css">
+  <!-- Load our custom CSS last so it overwrites the minified CSS above -->
+  <link rel="stylesheet" type="text/css" href="components/css/index.min.css">
```

Comment:
@cossou, I'll be monitoring the Assembly project for new bounties. Now that I have some experience with the project and understand the way it flows, It should be easier to pick up new aspects! :D